pkg/aws/throttle/defaults.go

```diff
@@ -0,0 +1,55 @@
+package throttle
+
+import (
+	"github.com/aws/aws-sdk-go/service/appmesh"
+	"github.com/aws/aws-sdk-go/service/servicediscovery"
+	"golang.org/x/time/rate"
+	"regexp"
+)
+
+// NewDefaultServiceOperationsThrottleConfig returns a ServiceOperationsThrottleConfig with default settings.
+func NewDefaultServiceOperationsThrottleConfig() *ServiceOperationsThrottleConfig {
+	return &ServiceOperationsThrottleConfig{
+		value: map[string][]throttleConfig{
+			appmesh.ServiceID: {
+				{
+					operationPtn: regexp.MustCompile("^Describe|List"),
+					r:            rate.Limit(40),
```

Comment:
How about converting these values in to consts with descriptive names?

Reply:
tbh, personally I think it's more clear to directly use them instead of using constants.
since all of these literals is used only once(there are multiple "1" or "8", but it's different per operationPtn). 
given they are only used once and we have many configuration per service/operation, it's clear to just list them in structs